### PR TITLE
Add imageGalleryElement component

### PR DESCRIPTION
**Pull request by shraddha-kesari**

### CODE REVIEW

File: components/image-gallery.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import { ResponsiveImage } from "./responsive-image";
+
+const ImageGalleryElement = (props) => {
+    const images = props.element['story-elements'].map(image => (
+      <figure key={image.id} className="story-element-image-gallery__image-container">
+        <ResponsiveImage slug={image["image-s3-key"]}
+                        metadata={image["metadata"]}
+                        aspect-ratio={[1,1]}
+                        defaultWidth= {480}
```
Comment (asimqt) [inline components/image-gallery.js]: aspectRatio?

**Comment by asimqt** [comment]
Please verify whether the prop `aspect-ratio` is okay or not. @munirajun

File: components/image-gallery.js (hunk 1 above)
Comment (munirajun) [inline components/image-gallery.js]: Shouldn't defaultWidth and widths be passed as props?

Comment (munirajun) [inline components/image-gallery.js]: before map can we filter image elements?

File: components/image-gallery.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import { ResponsiveImage } from "./responsive-image";
+
+const ImageGalleryElement = (props) => {
+    const images = props.element['story-elements'].map(image => (
+      <figure key={image.id} className="story-element-image-gallery__image-container">
+        <ResponsiveImage slug={image["image-s3-key"]}
```
Comment (asimqt) [inline components/image-gallery.js]: Pass down `aspecRatio` and `className` as props.

File: components/image-gallery.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import { ResponsiveImage } from "./responsive-image";
+
+const ImageGalleryElement = (props) => {
+    const images = props.element['story-elements'].map(image => (
```
Comment (asimqt) [inline components/image-gallery.js]: Try destructuring props and try defensive check for `element['story-elements']`

**Pull request merged**